overlay-app/src/css/Scoreboard.css

```diff
@@ -1,82 +1,144 @@
-.scoreboard {
+.scoreboard-wrapper {
 	position: absolute;
-	top: 0px;
+	top: 0;
+	left: 0;
+	right: 0;
+}
+
+.scoreboard {
+  position: relative;
+	margin: 0 auto;
 	transition: 1s;
-	left: calc(var(--scoreboard-w) - var(--scoreboard-hw));
-	height: var(--scoreboard-h);
+  
+  
 	--scoreboard-h: calc(var(--sh) / 15);
 	--scoreboard-w: calc(var(--sw) / 2);
 	--scoreboard-hw: calc(var(--scoreboard-w) / 2);
-  	width: var(--scoreboard-w);
+	--name-h: calc(var(--scoreboard-h) * 0.9);
+  height: var(--scoreboard-h);
 	opacity: 0.99;
+	--logo-size: calc(var(--name-h) + var(--tally-h));
+  
+  --center-w: calc(var(--sw) / 10);
+  --center-hw: calc(var(--center-w) / 2);
+  --score-w: calc(var(--sw) * .04);
+	--tally-h: calc(var(--sw) * .0078);
+
+  width: var(--scoreboard-w);
+}
+
+.scoreboard.has-logos {
```

Comment:
Made the scoreboard bigger when logos are present so the space for the rest wasn't compromised